packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -103,10 +106,14 @@ export class ProfilingPlugin {
 
         const issuerModule = compilation?.moduleGraph?.getIssuer(module)
 
-        const span = trace(
-          `build-module${moduleType ? `-${moduleType}` : ''}`,
-          issuerModule ? spans.get(issuerModule)?.id : compilerSpan.id
-        )
+        let span: Span
+
+        const spanName = `build-module${moduleType ? `-${moduleType}` : ''}`
+        if (issuerModule) {
+          span = spans.get(issuerModule)!.traceChild(spanName)
```

Comment:
It looks like this span is undefined causing this error: 

```
Failed to compile.

TypeError: Cannot read property 'traceChild' of undefined


> Build error occurred
/Users/jj/dev/vercel/next.js/packages/next/build/index.ts:636
      throw new Error('> Build failed because of webpack errors')
```